lib/local_authority/gateway/hif_returns_schema.rb

```diff
@@ -1331,6 +1331,58 @@ def execute
                   }
                 }
               },
+              currentFunding: {
+                type: "object",
+                title: " ",
```

Comment:
Having these as whitespace characters causes extra sections to appear in the form, even when the dependency hasn't been selected yet.
```suggestion
                title: "",
```